docs/design/UITesting.md

```diff
@@ -10,60 +14,36 @@ Appium relies on different implementations called `drivers` for each platform th
 * iOS       - [XCUITest](https://github.com/appium/appium-xcuitest-driver)
 * Android   - [UIAutomator2](https://github.com/appium/appium-uiautomator2-driver)
 
-## Creating a new test
-
-### The sample project
-
-The project that is used for UI Tests is located here: `src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj`
 
-There are two types of tests you can add, Issue and Gallery.
+## Creating a new test
 
 ### Adding a new Issue
 
 This will be the majority of new tests added which will be primarily for testing functionality and adding regression tests.
 
-You will need to create some kind of UI to test against, which will go in the Controls.Sample.UITests project. Create a new class and attribute it with `[Issue]` and derive from `TestContentPage`.
-
-Then in the Controls.AppiumTests project add a new class that derives from `_IssuesUITest` and add your test.
-
-You can use the example for the sample project [here](https://github.com/dotnet/maui/blob/main/src/Controls/tests/TestCases.HostApp/Issues/RefreshViewPage.cs) and the example for the corresponding test [here](https://github.com/dotnet/maui/tree/main/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/RefreshViewTests.cs).
-
-### Adding a GalleryPage
+You will need to create some kind of UI to test against, which will go in the Controls.TestCases.HostApp project. Create a new class within `src/Controls/tests/TestCases.HostApp/Issues` and attribute it with `[Issue]` and derive from `TestContentPage` (or `TestNavigationPage`, or `TestShellPage`, and so on).
 
-Gallery tests are to make it easier to run the same set of tests on controls, if you are creating a new control you would want to add a new gallery page.
-
-We have some base classes you can derive from to make setting this up easier: [CoreGalleryPage](https://github.com/dotnet/maui/blob/main/src/Controls/tests/TestCases.HostApp/CoreViews/CoreGalleryPage.cs) and [ContentViewGalleryPage](https://github.com/dotnet/maui/blob/main/src/Controls/tests/TestCases.HostApp/Elements/ContentViewGalleryPage.cs)
+Then in the Controls.TestCases.Shared.Tests project add a new class within this folder: `src/Controls/tests/TestCases.Shared.Tests/Tests/Issues`, or in one of the platform-specific projects. Have the class derive from `_IssuesUITest` and add your test.
```

Comment:
And then mention here how this is just a library project that runs NUnit tests via appium against the running `HostApp`

Maybe this can be a different section? 

The first one can be

#### Adding a Reproduction to the Test App
....

#### Adding a test to interact with the reproduction
....

I would remove the part about "or in one of the platform-specific projects". In theory we could do that but we don't actually do this currently. We only remove platform via if/defs that way things just all stay in one place